scripts/installer/windows/heat_post_process.ps1

```diff
@@ -0,0 +1,16 @@
+# Add the script to delete the pfs.exe component from the generated wxs file, which is conflicting with the pfs.exe
```

Comment:
May need to talk to Zhen on this. We might give this two different name instead of just delete.

Reply:
offline synced